src/Google/AccessToken/Verify.php

```diff
@@ -53,11 +53,11 @@ public function __construct(
       CacheItemPoolInterface $cache = null,
       $jwt = null
   ) {
-    if (is_null($http)) {
+    if (null === $http) {
       $http = new Client();
     }
 
-    if (is_null($cache)) {
+    if (null === $cache) {
```

Comment:
`null === $cache` does the same, but without operating on calls stack, so it simply [faster](https://hakre.wordpress.com/2011/03/29/php-is_null-vs-null/).
And de-facto using is_null already judged as a bad practice.